src/components/MapView/index.tsx

```diff
@@ -40,7 +40,7 @@ import {
   loadAvailableDates,
 } from '../../context/serverStateSlice';
 
-import appConfig from '../../config/prism.json';
+import { appConfig } from '../../config/config';
```

Comment:
What do you think of using `index.ts` inside the config folder so there isn't a double config?